Backend/app/rider-platform/rider-app/Main/spec/API/frfs-ticket.yaml

```diff
@@ -134,3 +142,39 @@ apis:
       auth: TokenAuth
       response:
         type: "[API.Types.UI.FRFSTicketService.FRFSTicketBookingStatusAPIRes]"
+
+  # Booking Soft Cancel
+  - POST:
+      endpoint: /frfs/booking/canCancel
```

Comment:
`booking_id` should be in path
`/frfs/booking/{booking_id}/canCancel`


Same below